### ISSUE TITLE
Term Request: Panstromal corneal lipid haze

### ISSUE THREAD

**Issue by daboe01** [obophenotype/human-phenotype-ontology#11750]
**Preferred term label:**
Panstromal corneal lipid haze

**Synonyms**
* Diffuse corneal stromal lipid opacification
* Panstromal corneal cholesterol haze
* Midperipheral panstromal haze

**Definition (free text, please give PubMed ID)**
Diffuse, progressive ground-glass opacification involving all depths of the corneal stroma, resulting from abnormal intracellular and extracellular deposition of esterified and unesterified phospholipids and cholesterol (PMID: 38359414).

**Parent term (use hpo.jax.org/app)**
Opacification of the corneal stroma (HP:0007759)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
* Schnyder corneal dystrophy (SCD) (ORPHA:98967, OMIM:121800)

**Your nano-attribution (ORCID)**
0000-0001-9042-0508
